README.md

```diff
@@ -6,11 +6,26 @@ Experimental TUI for [asdf](https://asdf-vm.com/)
 
 ## Installation
 
-- brew: `brew install mhanberg/tap/lazyasdf`
+### Using brew
+
+`brew install mhanberg/tap/lazyasdf`
+
+### Using git
+
+You need to install `python3-termbox` first, e.g.: `apt-get install python3-termbox`
```

Comment:
You do not need this, this looks to by python binding to termbox.

You do however need an unversioned python executable on your path.

I am not sure how to do this on Linux distros, but on my mac with homebrew I did

```bash
export PATH="$brew_prefix/opt/python@3.9/libexec/bin:$PATH"
```

libexec/bin is where homebrew installs the unversioned symlinks.

and to clarify, by unversioned i mean `python` vs `python3`.

Reply:
Thanks, I updated the readme to have this info instead of what I originally wrote